app/templates/main/select.html

```diff
@@ -49,7 +53,11 @@
             </label>
             <select class="govuk-select" id="to-year" name="to-year">
                 {% for item in params['to-year'] %}
-                    <option value="{{item}}">{{item}}</option>
+                    {% if loop.last %}
+                        <option value="{{item}}" selected>{{item}}</option>
+                    {% else %}
+                        <option value="{{item}}">{{item}}</option>
```

Comment:
Could we do this logic inline in the `<option>` element rather than repeating it in the else statement?